.github/workflows/example_dispatch.yml

```diff
@@ -0,0 +1,53 @@
+name: Manual Test Example
+on:
+  workflow_dispatch:
+    inputs:
+      example_directory:
+        type: string
+        description: example directory, separated by comma. For example, language/gpt, images/vit. Simply input language or simply gpt does not work.
+        required: true
+
+jobs:
+  manual_check_matrix_preparation:
+    name: Check the examples user want
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix-1.outputs.matrix }}
+    steps:
+    - id: set-matrix-1
+      env:
+        check_dir: ${{ inputs.example_directory }}
+      run: | 
+        IFS=','
+        all_mannual_check_dir=()
+        for cdi in $check_dir
+        do
+          all_mannual_check_dir+=("\"${cdi}\"")  
+        done
+        var=$( IFS=',' ; echo "${all_mannual_check_dir[*]}" )
+        var="[${var}]"
```

Comment:
`var` is a bad variable name. Meanwhile, what if the user inputs invalid diectory?